src/qilib/configuration_helper/adapters/m4i_instrument_adapter.py

```diff
@@ -26,10 +26,13 @@
 class M4iInstrumentAdapter(CommonInstrumentAdapter):
     """ Adapter for the Spectrum M4i digitizer."""
 
-    def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
-        super().__init__(address, instrument_name)
+    def __init__(self, address: str, instrument_name: Optional[str] = None,
+                 instrument_class: Optional[Type[Any]] = None) -> None:
         from qcodes_contrib_drivers.drivers.Spectrum.M4i import M4i
```

Comment:
Weren't we going to move the import out of the init? Or was this the issue that you talked about during the standup?

Reply:
yup, this is the stuff I raised during the standup. getting this import out of this method would require some more extensive modifications. we can schedule that work if it is necessary.

Reply:
Nah, don't think it is necessary